backend/sources/facebook/connector/src/main/java/co/airy/core/sources/facebook/Connector.java

```diff
@@ -78,13 +78,22 @@
 
             return List.of(KeyValue.pair(message.getId(), message), KeyValue.pair(getId(metadata).toString(), metadata));
         } catch (ApiException e) {
-            log.error(String.format("Failed to send a message to Facebook \n SendMessageRequest: %s \n Error Message: %s \n", sendMessageRequest, e.getMessage()), e);
```

Comment:
Even if we are returning the error now. It does not hurt to log it anyway

Reply:
Agreed, I will add them back in